Library/Formula/bzr-upload.rb

```diff
@@ -0,0 +1,30 @@
+class BzrUpload < Formula
+  desc "Bazaar plugin to incrementally upload changes to a dumb server"
+  homepage "https://launchpad.net/bzr-upload"
+  url "https://launchpad.net/bzr-upload/1.1/1.1.0/+download/bzr-upload-1.1.0.tar.gz"
+  sha256 "a48fc56d83114d9ab946cc358a5b33cb05e134787be135eb0a499317d6dec7fc"
+
+  bottle :unneeded
+  depends_on "bazaar"
```

Comment:
It's a nit but you keep doing it this way, so I'll point it out :smile:. The `bottle :xyz` line and the `depends_on` should always have an empty line between them.